### PR TITLE
Add request.referrer and request.origin shortcuts

### PR DESCRIPTION
**Pull request by yohanboniface**
Not sure we really need those, but I see that I use them quite often, so… :)

### CODE REVIEW

File: roll/io.py [hunk 1]
```diff
@@ -153,6 +153,15 @@ def content_type(self):
     def host(self):
         return self.headers.get('HOST', '')
 
+    @property
+    def referrer(self):
```
Comment (yohanboniface) [inline roll/io.py:157]: Do we also want `referer` as alias ?

Comment (davidbgk) [reply roll/io.py:157]: 👍

**Comment by yohanboniface** [comment]
This kind of usage: https://gitlab.com/eurordis/contact-database/-/blob/master/eurordis/__init__.py#L184

**Comment by davidbgk** [comment]
So now you'll be able to do: `hostname=request.origin or f"https://{request.host}",`?

**Comment by yohanboniface** [comment]
Yes, nothing more spectacular than this :p

**Pull request merged**